source/install-guide/installing-index.rst

```diff
@@ -10,30 +10,27 @@ If users want to analyze WFS data (See :ref:`analyzing_data`), an
 Manual installation
 -------------------
 
-Download Elasticsearch from https://www.elastic.co/downloads/elasticsearch
+Download Elasticsearch from https://www.elastic.co/downloads/past-releases/elasticsearch-7-4-2
 and unzip the file.
 
 
 .. code-block:: shell
+    
+    wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.4.2-linux-x86_64.tar.gz
+    tar xvfz elasticsearch-7.4.2-linux-x86_64.tar.gz
 
-    cd es/config
-    wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.4.2.tar.gz
-    tar xvfz elasticsearch-7.4.2.tar.gz
 
-
-Manually start and stop Elasticsearch using:
+Manually start Elasticsearch using:
 
 .. code-block:: shell
 
     elasticsearch-7.4.2/bin/elasticsearch
 
-
-Then create the default index:
-
+The elasticsearch startup will block the current terminal. From another terminal create the default index:
 
 .. code-block:: shell
 
-    cd es
+    cd elasticsearch-7.4.2
```

Comment:
Are you sure this is OK? From inside this directory `curl` can't find the `config/features.json`, `config/records.json` and `config/searchlogs.json` files, @gamesm.

Reply:
Hi @juanluisrp, you're correct those `curl` lines are wrong - those config files reside in the geonetwork package not the elasticsearch one.